src/frontend/citizen-portal/src/app/components/count-item-dispute-summary/count-item-dispute-summary.component.html

```diff
@@ -10,7 +10,9 @@
   <div class="col-lg-7">
     <small class="d-block">For this count I would like to</small>
     <strong>
-      <div *ngIf="count.__skip">
+      <div *ngIf="count.__skip 
+      || !(count.offenceDeclaration === 'Plea' || count.timeToPayRequest 
```

Comment:
careful with truthy checks see e.g. https://www.tektutorialshub.com/typescript/typescript-boolean/
its always best with boolean variables to use === true  or === false

Reply:
No problem, I will update it again after using the latest model